code/modules/mob/living/living.dm

```diff
@@ -677,23 +677,29 @@ default behaviour is:
 		requests.Remove(O)
 		qdel(O)
 		resisting++
+	var/resist_power = 1 // How easily the mob can break out of a grab
+	if(istype(src, /mob/living/carbon/human))
```

Comment:
As discussed in Discord, `istype(src, ...)` is a bad construct. Use a `mob/living/proc/get_resist_power()` and override that for the human type to reference `species.resist_mod`.